### DISCUSSION TITLE
Adding Github token?

### DISCUSSION THREAD

**Discussion by eried** [akopachov/flipper-zero_authenticator#86] [Q&A] [answered]
hi!

What settings do I need to add github? I tried with all the algorithms and my timezone (+1) for norway and 0 and 2,3,4

But the code seems to be always wrong. The clock is correct too. I just copy this code: 
![image](https://user-images.githubusercontent.com/1091420/222926413-530c276e-f8bc-4d00-b981-cd87079881f1.png)

pasting it in the web CLI 

What I am doing wrong?

**Comment by akopachov** [comment, marked as answer]
Hey, so you did all the right things. For Github there is no need to use custom algo, it should be default SHA1.
Could you please share the output of `totp tz` command to make sure it is set correctly?
Also would you mind sharing `totp ls` command output just for that specific Github account you added? (please do not include any other accounts into screenshot).

I tried to setup Github account myself and it does work just fine, so we just need to figure out what option is not set or so.

**Reply by eried** [reply]
Hi! ok, so what I discovered is sometimes when first add, it is not properly generated. After reopening the app is fine.

So what I recommend is checking with https://www.verifyr.com/en/otp/check vs flipper :)
